routes/rooms.go

```diff
@@ -127,12 +127,56 @@ func CreateEmergencyRoom(ctx *gin.Context) {
 // @Produce    json
 // @Param      authorization                   header      string                true    "Bearer: <TOKEN>"
 // @Param      id                              path        string                true    "Emergency Room's ID"
-// @Success    200                             {object}    GetSingleERResponse
-// @Failure    501                             {object}    HTTPErrorResponse
+// @Param      emergency-room                  body        PutERRequest          true    "ER to update"
+// @Success    200
```

Comment:
Should we return the new state?

Reply:
No, I think you should fetch it using the GET-endpoint.